pennylane/ops/channel.py

```diff
@@ -267,6 +267,11 @@ class DepolarizingChannel(Channel):
     where :math:`p \in [0, 1]` is the depolarization probability and is equally
     divided in the application of all Pauli operations.
 
+    .. note::
```

Comment:
I think it would be good to point out explicitly that there are different conventions and we adopt this one, while also providing a link to a paper/book that also uses the same convention as us